Documentation

I can't find the docs on the cmake build folder. Someone could help me?
I used the steps from the README.md

Thanks.

```
lucasamparo@amparo-N8xxEP6:build (master)$ make doc
/bin/sh: 1: cd: can't cd to /doc
doc/CMakeFiles/doc.dir/build.make:57: recipe for target 'doc/CMakeFiles/doc' failed
make[3]: *** [doc/CMakeFiles/doc] Error 2
CMakeFiles/Makefile2:690: recipe for target 'doc/CMakeFiles/doc.dir/all' failed
make[2]: *** [doc/CMakeFiles/doc.dir/all] Error 2
CMakeFiles/Makefile2:697: recipe for target 'doc/CMakeFiles/doc.dir/rule' failed
make[1]: *** [doc/CMakeFiles/doc.dir/rule] Error 2
Makefile:331: recipe for target 'doc' failed
make: *** [doc] Error 2
```

Both. Same results.
```
lucasamparo@amparo-N8xxEP6:doc (master)$ cd ..
lucasamparo@amparo-N8xxEP6:build (master)$ make doc
/bin/sh: 1: cd: can't cd to /doc
doc/CMakeFiles/doc.dir/build.make:57: recipe for target 'doc/CMakeFiles/doc' failed
make[3]: *** [doc/CMakeFiles/doc] Error 2
CMakeFiles/Makefile2:690: recipe for target 'doc/CMakeFiles/doc.dir/all' failed
make[2]: *** [doc/CMakeFiles/doc.dir/all] Error 2
CMakeFiles/Makefile2:697: recipe for target 'doc/CMakeFiles/doc.dir/rule' failed
make[1]: *** [doc/CMakeFiles/doc.dir/rule] Error 2
Makefile:331: recipe for target 'doc' failed
make: *** [doc] Error 2
lucasamparo@amparo-N8xxEP6:build (master)$ sudo make doc
[sudo] senha para lucasamparo: 
/bin/sh: 1: cd: can't cd to /doc
doc/CMakeFiles/doc.dir/build.make:57: recipe for target 'doc/CMakeFiles/doc' failed
make[3]: *** [doc/CMakeFiles/doc] Error 2
CMakeFiles/Makefile2:690: recipe for target 'doc/CMakeFiles/doc.dir/all' failed
make[2]: *** [doc/CMakeFiles/doc.dir/all] Error 2
CMakeFiles/Makefile2:697: recipe for target 'doc/CMakeFiles/doc.dir/rule' failed
make[1]: *** [doc/CMakeFiles/doc.dir/rule] Error 2
Makefile:331: recipe for target 'doc' failed
make: *** [doc] Error 2
lucasamparo@amparo-N8xxEP6:build (master)$ cd doc
lucasamparo@amparo-N8xxEP6:doc (master)$ make doc
/bin/sh: 1: cd: can't cd to /doc
doc/CMakeFiles/doc.dir/build.make:57: recipe for target 'doc/CMakeFiles/doc' failed
make[3]: *** [doc/CMakeFiles/doc] Error 2
CMakeFiles/Makefile2:690: recipe for target 'doc/CMakeFiles/doc.dir/all' failed
make[2]: *** [doc/CMakeFiles/doc.dir/all] Error 2
CMakeFiles/Makefile2:697: recipe for target 'doc/CMakeFiles/doc.dir/rule' failed
make[1]: *** [doc/CMakeFiles/doc.dir/rule] Error 2
Makefile:172: recipe for target 'doc/CMakeFiles/doc.dir/rule' failed
make: *** [doc/CMakeFiles/doc.dir/rule] Error 2
lucasamparo@amparo-N8xxEP6:doc (master)$ sudo make doc
/bin/sh: 1: cd: can't cd to /doc
doc/CMakeFiles/doc.dir/build.make:57: recipe for target 'doc/CMakeFiles/doc' failed
make[3]: *** [doc/CMakeFiles/doc] Error 2
CMakeFiles/Makefile2:690: recipe for target 'doc/CMakeFiles/doc.dir/all' failed
make[2]: *** [doc/CMakeFiles/doc.dir/all] Error 2
CMakeFiles/Makefile2:697: recipe for target 'doc/CMakeFiles/doc.dir/rule' failed
make[1]: *** [doc/CMakeFiles/doc.dir/rule] Error 2
Makefile:172: recipe for target 'doc/CMakeFiles/doc.dir/rule' failed
make: *** [doc/CMakeFiles/doc.dir/rule] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,7 +80,7 @@ target_include_directories(tinyply PUBLIC
         $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/tinyply-src/source>)
 
 # Building Targets
-set(TEASER_ROOT ${CMAKE_CURRENT_LIST_DIR})
+set(TEASERPP_ROOT ${CMAKE_CURRENT_LIST_DIR})
 add_subdirectory(teaser)
 
 if (BUILD_TESTS)

--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -1,9 +1,9 @@
-project(teaser_doc)
+project(teaserpp_doc)
 
 set_directory_properties(PROPERTIES EXCLUDE_FROM_ALL TRUE)
 
-set(TEASER_DOXY_PROJECT_NAME             "TEASER")
-set(TEASER_DOXY_INPUT                    "\"${TEASER_SOURCE_DIR}/teaser\" \"${TEASER_SOURCE_DIR}/doc\"")
+set(teaserpp_DOXY_PROJECT_NAME             "teaserpp")
+set(teaserpp_DOXY_INPUT                    "\"${teaserpp_SOURCE_DIR}/teaser\" \"${teaserpp_SOURCE_DIR}/doc\"")
 
 configure_file(
         ${CMAKE_CURRENT_SOURCE_DIR}/Doxyfile.in
@@ -12,4 +12,4 @@ configure_file(
 
 add_custom_target(doc ALL
         COMMAND doxygen
-        WORKING_DIRECTORY ${TEASER_BINARY_DIR}/doc)
+        WORKING_DIRECTORY ${teaserpp_BINARY_DIR}/doc)

--- a/doc/Doxyfile.in
+++ b/doc/Doxyfile.in
@@ -32,7 +32,7 @@ DOXYFILE_ENCODING      = UTF-8
 # title of most generated pages and in a few other places.
 # The default value is: My Project.
 
-PROJECT_NAME           = "${TEASER_DOXY_PROJECT_NAME}"
+PROJECT_NAME           = "${teaserpp_DOXY_PROJECT_NAME}"
 
 # The PROJECT_NUMBER tag can be used to enter a project or revision number. This
 # could be handy for archiving the generated documentation or if some version
@@ -58,7 +58,7 @@ PROJECT_LOGO           =
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = "${TEASER_BINARY_DIR}/doc"
+OUTPUT_DIRECTORY       = "${teaserpp_BINARY_DIR}/doc"
 
 # If the CREATE_SUBDIRS tag is set to YES then doxygen will create 4096 sub-
 # directories (in 2 levels) under the output directory of each output format and
@@ -790,7 +790,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ${TEASER_DOXY_INPUT}
+INPUT                  = ${teaserpp_DOXY_INPUT}
 
 # This tag can be used to specify the character encoding of the source files
 # that doxygen parses. Internally doxygen uses the UTF-8 encoding. Doxygen uses

